docs/overview/changelog.md

```diff
@@ -61,8 +59,10 @@ _Released 2025-07-29_
 - Reworked Keyboard shortcuts to better support non-Latin keyboard
   locales. Now, keyboard shortcuts are entirely based on the key
   pressed, not what character it is mapped to.
+- Added SCIM support for syncing group membership.
 - The LDAP integration now supports automatically handling changes in
   user email addresses, when provided with a unique ID for each user.
+- Added support for Debian 13 trixie.
```

Comment:
Should "Trixie" be capitalized?

Reply:
https://www.debian.org/News/2025/20250809 does not capitalize, but does put it in quotes, so I guess we can do that.